dlt/common/destination/reference.py

```diff
@@ -55,6 +69,16 @@
 
 DEFAULT_FILE_LAYOUT = "{table_name}/{load_id}.{file_id}.{ext}"
 
+try:
+    from dlt.common.libs.pandas import DataFrame
```

Comment:
I need these in the reference for the DataAccess classes. I could also type those as any, but It would be nicer if the user had fully typed results here.

Reply:
Ok. I'm assuming i still need to catch the MissingDependencyException exceptions in case a user uses mypy on a project and does not need these libs, right?